shopinvader_product_brand/models/product_brand.py

```diff
@@ -6,16 +6,10 @@
 
 
 class ProductBrand(models.Model):
-    _inherit = "product.brand"
+    _inherit = ["product.brand", "seo.title.mixin", "abstract.url"]
 
-    shopinvader_bind_ids = fields.One2many(
-        "shopinvader.brand",
-        "record_id",
-        string="Shopinvader Binding",
-        context={"active_test": False},
-    )
-    active = fields.Boolean(default=True, inverse="_inverse_active")
-
-    def _inverse_active(self):
-        brands = self.filtered(lambda p: not p.active)
-        brands.mapped("shopinvader_bind_ids").write({"active": False})
+    active = fields.Boolean(default=True)
+    sequence = fields.Integer()
+    meta_description = fields.Char()
+    meta_keywords = fields.Char()
+    short_description = fields.Html()
```

Comment:
`description` is defined on addon `product_brand` but is a `Text` field. In v14 we convert it to an `Html` field but I think that, following a discussion on another PR, this is a good thing to replace `Html` fields by `Text` fields.
In this case we could do the same for `short_description`